lib/kubernetes-deploy/runner.rb

```diff
@@ -104,7 +105,9 @@ def run(verify_result: true, allow_protected_ns: false, prune: true)
 
       if deploy_has_priority_resources?(resources)
         @logger.phase_heading("Predeploying priority resources")
-        predeploy_priority_resources(resources)
+        ::StatsD.measure('duration.priority_resources', tags: %W(namespace:#{@namespace} sha:#{ENV['REVISION']})) do
```

Comment:
I see that it measures the duration of predeploy resources and the duration of verification separately, but from the dashboard perspective I think that the most valuable metric would be the total deploy time. Should we measure it as well?